_dashboards/selfhost-maps-server.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: default
+title: Self-host maps server
+nav_order: 6
+---
+
+# Self-host maps server
+
+The self-host maps server for OpenSearch Dashboards allows users to access the default maps service on air-gapped environments. The server provides a map manifest with map tiles and vectors URL, map tiles URL, and map vectors URL that are compatible with OpenSearch.
+
+The following steps walk through setting up and using the self-host maps server with OpenSearch Dashboards.
+
+You can access the `maps-server` image via the OpenSearch official [Docker Hub repository](https://hub.docker.com/u/opensearchproject).
+{: .note}
+
+## Pull Docker image
+
+Open your terminal and run the following command:
+
+`docker pull opensearch/opensearch-maps-server`
+
+## Set up the server
+
+You must set up the map tiles before running the server. You have two setup options: 
+
+### Option 1: Use the OpenSearch-provided maps service tiles set
```

Comment:
Here this is "tiles set", but we use "tile set" below. Let's make consistent, as appropriate.